cranelift-codegen/meta/src/cdsl/types.rs

```diff
@@ -299,7 +299,7 @@ pub struct LaneTypeIterator {
 
 impl LaneTypeIterator {
     /// Create a new lane type iterator.
-    fn new() -> Self {
+    pub fn new() -> Self {
```

Comment:
You can use `ValueType::all_lane_types()` as call sites instead of making this public.